src/common/ActivityInfo/ActivityInfo.tsx

```diff
@@ -85,6 +98,25 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
                 </ul>
               </Card.Body>
             </Card>
+            <br />
+            <Card>
+              <Card.Body>
+                <Card.Title>Join the Discussion:</Card.Title>
+                <Card.Text>
+                  Join our slack to discuss with presenters, mentors, and other
+                  TAMU Datathon 2020 attendees.
+                  <br />
+                  {channelLink}
+                </Card.Text>
+                <Button
+                  href={"#slack-link-here"}
```

Comment:
The link is: https://join.slack.com/t/tamudatathon2020/shared_invite/zt-fnyi8tdv-Tm1ArAaznmVd9cXDaglq9Q